libpod/container_internal_linux.go

```diff
@@ -607,10 +607,11 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 
 	availableUIDs, availableGIDs, err := rootless.GetAvailableIDMaps()
 	if err != nil {
-		return nil, err
+		// The kernel-provided files only exist if user namespaces are supported
```

Comment:
You should be checking the specific error here. Probably ENOENT.  Theoretically you could get other errors line EPERM. Also probably should logrus.Debug this.

Reply:
Thanks for the quick feedback. Added check for specific error as well as a debug message.